automation-orchestrator/automation_orchestrator_api.py

```
@@ -0,0 +1,228 @@
+    try:
+        job_queue.put(req)
+
+        print(f"[QUEUE] Added job: {req.raw_path} | Queue size: {job_queue.qsize()}")
+
+        return {
+            "status": "queued",
+            "message": "ETL job added to queue",
+            "raw_path": req.raw_path
+        }
+    
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)